### DISCUSSION TITLE
Check toggle menu in finish mama project

### DISCUSSION THREAD

**Discussion by Vivioza5** [open-tutorials/cypress#39] [Ideas]
Прохожу туториал: заканчиваем мама проект, выполняю в части написания кода для просмотра случайной  статьи.
В описании тест кейса в текстовом формате пристутствует пункт 3:

3. **Global Feed** item should be active in feed toggle menu

написал код, начал сверять с эталоном у увидел, что там этот пункт в реализации отсутствует, есть предложение, либо убрать этот пункт из текстовой версии тест кейса либо добавить:

```js
cy.get('.feed-toggle ')
    .find('a[class$="active"]')
    .should('be.visible')
```

**Comment by Vladimir-Tarasov** [comment]
Cпасибо за замечание. Проверим

**Comment by breslavsky** [comment]
Спасибо за вашу помощь проекту.

Добавил проверну вот так

```js
// TODO: [data-cy=feed-menu] a[data-cy=global]
cy.get('.feed-toggle ul > li:nth-child(2) a')
    .click()
    .should('have.class', 'active');
```
